data/build.gradle

```diff
@@ -28,6 +32,12 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+        kotlinOptions {
+            jvmTarget = '1.8'
+            freeCompilerArgs += "-opt-in=org.mylibrary.OptInAnnotation"
```

Comment:
Are you sure it's correct? Note the package :)

Reply:
And I changed it in module `app` since I had the same problem